### DISCUSSION TITLE
Consulta repo de ejemplos creacionales

### DISCUSSION THREAD

**Discussion by RaniAgus** [dds-jv/foro#349] [Consultas de objetos] [answered]
¡Buenas! Una consulta: estaba viendo el [repo de ejemplos creacionales](https://github.com/dds-utn/ejemplos-creacionales) y entre los ejemplos de Builder vi que hay uno de "Fechas en Java" que está vacío.

¿Se refiere a la clase [Calendar.Builder](https://docs.oracle.com/javase/8/docs/api/java/util/Calendar.Builder.html)? Quizás antes estaba en el ejemplo pero como entiendo que `Calendar` se dejó de usar en favor de `LocalDateTime` se puede haber quitado y quedó solo el título :P

También en un principio pensé que podría ser [LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html) en sí, pero las formas que vi de crear fechas son usando métodos (sin pasos intermedios), como `LocalDate.now()` o `LocalDateTime.of(year, month, day, hour, minute)`, por lo que me suena a que es más un ejemplo de Factory Method, ¿no?

Saludos

**Comment by gastonprieto** [comment, marked as answer]
Buenas !

Estuvimos investigando y no recordamos que era, así que directament el ejemplo, puede ser que se refiera a ese, pero puede que haya quedado desactualizado por algun motivo

**Reply by RaniAgus** [reply]
Ahhhh ok, buenísimo. Gracias Gastón 🤗
¡Saludos!
